test/e2e/scenario/policy-report/chainsaw-test.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
```

Comment:
Should there be a `---` somewhere here?

Reply:
Why? If you are thinking about the multi-document separator, we can fix it in another PR.

Reply:
Yeah most of if not all the other files have the document start dahses